src/console_cmds.cpp

```diff
@@ -2412,6 +2525,8 @@ void IConsoleStdLibRegister()
 	IConsoleCmdRegister("ban",             ConBan, ConHookServerOnly);
 	IConsoleCmdRegister("unban",           ConUnBan, ConHookServerOnly);
 	IConsoleCmdRegister("banlist",         ConBanList, ConHookServerOnly);
+	IConsoleCmdRegister("banlist_load", ConBanListLoad, ConHookServerOnly);
+	IConsoleCmdRegister("banlist_save", ConBanListSave, ConHookServerOnly);
```

Comment:
Follow the indentation of the surrounding lines, so the function pointers are aligned in the "table".